### DISCUSSION TITLE
New contribution proposal: Icons for media editing

### DISCUSSION THREAD

**Discussion by dGoligorsky** [Shopify/polaris-react#11792] [Contribution proposals]
### TL;DR

There's a few icons in use for Magic Media Generation that aren't in Polaris. That's the feature where product images can have the background removed and replaced with a solid color or a generated background image.
[Figma for the unflattened icons](https://www.figma.com/file/oeKyR0kuHIMktdVjdLHJvy/branch/xwAGmxX5vDpIT7DvbsJWH3/Polaris-Icons-(unflattened)?type=design&node-id=276-62&mode=design&t=r49u2jaqASUp2iFw-11)

- Crop
- Magic image
- Rotate left
- Rotate right
- Mirror horizontal
- Mirror vertical
- Lighting/Brightness

### Champion

David Goligorsky

### Stakeholders

Arda Karaçizmeli (Magic Media web, he is aware), Joe Thomas (icon lord)

### Estimated timeline

I'll flatten and optimize the svgs and do a PR once we're happy w design.

### What problem are you solving?

Just an icon contribution so we're using the desktop/mobile Polaris icons.

### How does your solution improve Polaris?

There's some non-componentized svg icons floating around, they should be Polaris icons.

### Have you tested and verified your solution?

Nope

### What is the scope of work?

Will do updates in Figma UI Kit and PR for the icon svg and yml

### What is the impact on existing implementations, and do you have a plan to address it?

I'll get mobile and web to flip from hardcoded svgs to using the components.

### Are there any known blockers or risks?

Nope

**Comment by dGoligorsky** [comment]
Got the [Figma branch in the UI Kit approved](https://www.figma.com/file/fVIazfJNe3AOJTJmBKgTO9/branch/0HsV7BFZSVjsfMVvxe1lSb/Polaris-Gen-3-Icons?type=design&mode=design&t=uyBLhqDq6u14QzNE-11) and will go forward with making a PR.
